VisualPinball.Unity/VisualPinball.Unity.Editor/Materials/MaterialEditor.cs

```diff
@@ -0,0 +1,330 @@
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Mathematics;
```

Comment:
Doesn't seem to be used and we don't use Unity's legacy math lib, so it's red in my IDE